packages/tests/tests/base_protoco.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { Peer } from "@libp2p/interface/peer-store";
+import type { Tag } from "@libp2p/interface/peer-store";
+import { createSecp256k1PeerId } from "@libp2p/peer-id-factory";
+import { LightNode, Tags } from "@waku/interfaces";
+import { createLightNode } from "@waku/sdk";
+import { expect } from "chai";
+
+describe("getPeers function", function () {
+  let waku: LightNode | undefined;
+
+  beforeEach(async function () {
+    waku = await createLightNode();
+  });
+
+  it("should return all peers when numPeers is 0", async function () {
+    const peer1 = await createSecp256k1PeerId();
+    const peer2 = await createSecp256k1PeerId();
+    const peer3 = await createSecp256k1PeerId();
+
+    const mockPeers = [
+      {
+        id: peer1,
+        tags: new Map<string, Tag>([[Tags.BOOTSTRAP, { value: 100 }]])
+      },
+      {
+        id: peer2,
+        tags: new Map<string, Tag>([[Tags.BOOTSTRAP, { value: 100 }]])
+      },
+      {
+        id: peer3,
+        tags: new Map<string, Tag>([[Tags.BOOTSTRAP, { value: 100 }]])
+      }
+    ] as unknown as Peer[];
+
+    const result = await (waku?.lightPush as any).filterPeers(mockPeers, 0, 0);
+    expect(result.length).to.deep.equal(mockPeers.length);
+  });
+
+  it("should return all peers, except bootstrap, when numPeers is 0 & maxBootstrap is defined", async function () {
```

Comment:
the title does not match the test as the expectation is that it returns one bootstrap peer